gensrc/thrift/InternalService.thrift

```diff
@@ -368,64 +370,6 @@ struct TTransmitDataResult {
   4: optional Types.TPlanNodeId dest_node_id
 }
 
-struct TTabletWithPartition {
-    1: required i64 partition_id
-    2: required i64 tablet_id
-}
-
-// open a tablet writer
-struct TTabletWriterOpenParams {
```

Comment:
What are these deleted data structures for? Will there be compatibility issues during rolling upgrades?